test/bin/scenario.sh

```diff
@@ -178,6 +187,7 @@ launch_vm() {
     fi
 
     enable_ip_access "${full_vmname}" "${ip}"
+    trap "sos_report" EXIT
```

Comment:
For scenarios that have multiple VMs, this will set up the trap multiple times. You could do it in `action_create` instead, so that it only happens once.

Reply:
Moved it to the top, as it may be useful from the beginning.